### DISCUSSION TITLE
Natural color with LOGB=113 and TONE=3

### DISCUSSION THREAD

**Discussion by lyricsdude** [gopro/labs#617] [General]
Hi,

I used Natural color with LOGB=113 and TONE=3.

Basically I want to achieve natural colors with a flat gopro log curve. I'm happy with it, but only the noise is harsher. How can I do this better?

Would be nice to have an option COLOR=1, COLOR=2 ...

**Comment by dnewman-gpsw** [comment]
if you request cF - color Flat, that is LOGB=113, with tone-mapping disabled, the color is to "spec" Rec709. The using Natural with LOGB and TONE=3 is a bit weird. The result will have more saturation, but may be under-exposed, likely why you more need noise. This combination is not clearly defined, as Natural uses tone-mapping and a Rec09 transfer (both you have disabled.)

**Reply by lyricsdude** [reply]
thanks for your reply. it would be nice to have natural colors and to create our own contrast curve. that's not possible?

**Reply by dnewman-gpsw** [reply]
Those request are at odds, as color saturation relates to contrast, and log curves lower the contrast.  Log curves are not intended for direct usage, but for color correction, as the curve is not optimized for display.  A log curve "corrected" to a display curve, will automatically make the colors more saturated.

**Reply by lyricsdude** [reply]
but the basic colors from the flat profile and WIDE=1 with any curve applied to it can't made look "natural" I mean. that's why I'm looking for a way to use the "natural" colors and apply my own curve.

**Reply by dnewman-gpsw** [reply]
WIDE is a different gamut, that will require a LUT.   Anything can be roughly made to look like anything else, just work (someone calibration tools.)  As Natural uses tone-mapping, it is a little harder to match, and editing tool rarely include tone-mapping controls

**Reply by lyricsdude** [reply]
i try to say it different. i'm looking for a flat to rec709 color transform. and then apply my own contrast or tone mapping.

**Reply by dnewman-gpsw** [reply]
As both curves are known, the transfer function conversion is straight forward.  Math is here  https://www.desmos.com/calculator/gzfzbrqpjh

This is a technical LUT (nothing creative), that convert Log113 (Protune Flat) to Rec709.  Stock Rec 709 often looks a bit dark, so that is where you get your creative input.  The colors will be restore to default saturation.
[GoProFlat2Rec709_33.zip](https://github.com/gopro/labs/files/13168820/GoProFlat2Rec709_33.zip)

**Reply by lyricsdude** [reply]
thanks but these are the same colors as flat profile, just normalized. the "natural" setting has totally different colors. that's what i'm after...

**Reply by lyricsdude** [reply]
blues are wayyy too blue and reds look orange

**Reply by dnewman-gpsw** [reply]
Natural has is own creative GoPro look, you can use post tools to match.  I only provided a technical LUT, not a creative one (more work).

**Reply by lyricsdude** [reply]
that creative look how can i achieve that? You have a LUT for that please? I would love to have those exact colors

**Reply by dnewman-gpsw** [reply]
It can be created, I have not done that recently.  There are some new LUT for HERO12 coming, when released I will back-convert them for Protune Flat.

**Reply by lyricsdude** [reply]
thank you so much. looking forward to that

**Reply by lyricsdude** [reply]
i'm playing around with TONE=3 right now. If I bump the exposure +1.0 in davinci the footage looks good. But lots of highlights look washed out. Can't you fix the exposure error on this? That would be great :)

Btw is it fixed on Gopro 12? Because it says Gopro 11 has this error...

**Reply by dnewman-gpsw** [reply]
Same on H12.  This is not technically an exposure error, is the way the camera works.  It under-exposes, this using GTM/LTM to get the look with rolled off highlights.

**Reply by lyricsdude** [reply]
But TONE=3 I can replicate in Davinci with a transform. Just not the creative look of the colors... How can I get the creative look colors? It's kinda strange that we can't use the standard gopro colors that we get straight out of the camera and apply shadows and highlights to our liking. Other camera companies all have that with LUT's and transforms.

**Reply by dnewman-gpsw** [reply]
I'm sorry, I don't think I understand your ask.  We have LUTs for GP-Log to natural, but not for every Labs setting.

**Reply by lyricsdude** [reply]
It would be great a new option like "TONE=4 for example" with adjusted normal exposure. Can you try to implement that?

**Reply by dnewman-gpsw** [reply]
I don't think that can be done, at least wouldn't look any different than something like  oMEVBS=2oMTONE=3

**Reply by lyricsdude** [reply]
adjusting the exposure doesn't work on TONE=3 unfortunately
